docs/basic-features/eslint.md

```diff
@@ -85,6 +85,7 @@ Next.js provides an ESLint plugin, [`eslint-plugin-next`](https://www.npmjs.com/
 | ✔️  | [@next/next/google-font-display](/docs/messages/google-font-display.md)                                                     | Enforce font-display behavior with Google Fonts.                                                   |
 | ✔️  | [@next/next/google-font-preconnect](/docs/messages/google-font-preconnect.md)                                               | Ensure `preconnect` is used with Google Fonts.                                                     |
 | ✔️  | [@next/next/inline-script-id](/docs/messages/inline-script-id.md)                                                           | Enforce `id` attribute on `next/script` components with inline content.                            |
+| ✔️  | [@next/next/next-key-head](/docs/messages/next-key-head.md)                                                                 | Enforce `key` attribute in children components of `next/head`.                                     |
```

Comment:
This shouldn't have a checkbox since the default is `off` right?

Reply:
Hmm, so should I remove this row? 
or maybe anything else?